tests/acceptance/routeable-engine-demo-test.js

```diff
@@ -277,6 +277,40 @@ test('transitionTo works properly within parent application', function(assert) {
   });
 });
 
+test('transitionToExternal transitions to the parent application from within an engine', function(assert) {
```

Comment:
We should also note that this test verifies the return value of `transitionToExternal`. Similarly for the `replaceWithExternal` test below.